src/site/content/en/mini-apps/mini-app-markup-styling-and-scripting/index.md

```diff
@@ -270,6 +271,47 @@ tt.canIUse("request.success.data");
   The next chapter introduces [mini app components](/mini-app-components/).
 {% endAside %}
 
+## Updates
+
+Mini apps do not have a standardized update mechanism ([discussion around potential standardization](https://github.com/w3c/miniapp-packaging/issues/28)).
+All mini app platforms have a backend system, which allows mini app developers to upload new versions of
+their mini apps. A super app then uses that backend system to check for and download updates. Some super apps
+perform updates entirely in the background, without any way for the mini app itself to influence the update
+flow. Other super apps give more control to the mini apps themselves.
+
+The following paragraphs describe [WeChat's update mechanism for mini apps](https://developers.weixin.qq.com/miniprogram/dev/framework/runtime/update-mechanism.html)
+in more detail. WeChat checks for available updates in the following two scenarios:
+
+1. WeChat will regularly check for updates of recently used mini apps as long as WeChat is running. If an
+   update is found, the update is downloaded, and synchronously applied the next time the user cold-starts the
+   mini app. "Cold starting" a mini apps occurs when the mini app was not currently running when the user
+   opened it (WeChat force-closes mini apps after being in the background for 5 minutes).
+2. WeChat also checks for updates when a mini app is cold-started. For mini apps that the user has not opened
+   "for a long time", the update is checked for and downloaded synchronously. While the update is downloading,
```

Comment:
```suggestion
   for a long time, the update is checked for and downloaded synchronously. While the update is downloading,
```

You don't use air quotes further down, where you say "recently". I think it's fine to remain vague at both locations.

Reply:
Fair!